documents/wg-application.md

```diff
@@ -0,0 +1,93 @@
+# CUDA WG - draft
+
+## What value do you want to bring to the project?
+
+The Working Group is an attempt to combine efforts to bring reliable support of writing safe GPGPU CUDA code with Rust.
+The work that will be done is supposed to clear the path for other GPGPU platforms in future.
+
+## Why should it be put into place?
+
+First steps to basic support of the feature have already been made by individual developers.
+Regardless of recent progress, there are still a lot of questions about safety and soundness of SIMT (Single Instruction, Multiple Threads) code and they require a lot of collaboration to find solutions.
+
+## What is the working group about?
+
+We want to work together on getting a solid foundation for writing CUDA code.
+Getting a safe and production-ready development experience on Stable Rust is our primary goal!
+
+Also, currently a major obstacle for developing GPGPU applications and algorithms in Rust is a lack of learning resources.
+We plan to solve the "documentation debt" with a broad range of tutorials, examples and references.
+
+## What is the working group not about?
+
+The WG is not focused on promoting or developing "standard" frameworks.
+Instead, we want to provide basic and reliable support of the feature and inspire the community to start using it.
+This should lead to experimenting with different approaches on how to use it and creating awesome tooling.
+
+## Is your WG long-running or temporary?
+
+In our current vision, the WG should live until we fulfill our goals.
+
+In the end, we hope the WG will evolve into another one to cover similar topics:
+to support other GPGPU platforms or to create higher-level frameworks to improve end-to-end experience based on community feedback.
+
+## What is your long-term vision?
+
+Having a reliable and safe CUDA development experience is our ultimate goal.
+This should include:
+
+* Getting `nvptx64-nvidia-cuda` to a [Tier 2](https://forge.rust-lang.org/platform-support.html) support state.
+* Test infrastructure on a real hardware and running the tests during Rust CI process.
+* Rich documentation, references, tutorials and examples.
+* A broad set of new compile errors, warnings and lints for SIMT execution model to avoid pitfalls and ensure code soundness.
+
```

Comment:
This section feels like too much. I'd just keep it short and just mention "Achieving all Milestones in our Roadmap."